src/Controller/DataFlowTransformsController.php

```diff
@@ -67,14 +67,13 @@ public function preview(Request $request, DataFlow $dataFlow, DataTransform $tra
         $totalNumberOfSteps = $dataFlow->getTransforms()->count();
         $steps = null !== $transform ? $transform->getPosition() + 1 : 0;
 
-        $results = $this->dataFlowManager->run($dataFlow, [
+        $run = $this->dataFlowManager->run($dataFlow, [
```

Comment:
$run is not the best name for this variable

Reply:
Renamed to `result`.